garage.com/src/main/java/ConvertibleCar.java

```diff
@@ -1,6 +1,20 @@
 public class ConvertibleCar extends Car implements ICar {
 
+	private boolean getIsLowerTop;
```

Comment:
These field names should not start with 'get'. They are not verbs. Rename is isTopLowered. You can also combine the two fields.